src/features/auth/services/login.service.ts

```diff
@@ -1,9 +1,9 @@
-import { BACKEND_V1_URL } from "app/config";
+import { BACKEND_V2_URL } from "app/config";
 import { BackendError } from "app/exceptions";
 import axios from "axios";
 import { Role } from "core/users/types";
 
-const URL = `${BACKEND_V1_URL}/auth/login`;
+const URL = `${BACKEND_V2_URL}/auth/login`;
```

Comment:
Es uno de mis primeros comentarios
```suggestion
const URL = `${BACKEND_V1_URL}/auth/login`;
```

Reply:
Listo